### ISSUE TITLE
Broken link in README.TXT

### ISSUE THREAD

**Issue by benjub** [metamath/metamath-exe#69]
README.TXT contains the broken link http://utopia.knoware.nl/~hlub/uck/rlwrap/

Maybe replace it with https://github.com/hanslub42/rlwrap (or https://manpages.org/rlwrap or https://linux.die.net/man/1/rlwrap) ?

More generally, there may be alternatives ? (Personally, I always use rlwrap with metamath, although it has the annoying feature of deleting the prompt message of previous instructions.)

Also: shouldn't README.TXT be renamed README.md (and reformatted to use markdown) ?

**Comment by david-a-wheeler** [comment]
Yes, we should switch the link. The URL https://manpages.org/rlwrap seems likely to be safe.

We don't *need* to switch it to markdown, though we certainly can.
